lib/html/includes.tmpl

```diff
@@ -142,6 +142,9 @@
 <script id="linksTemplate" type="text/x-dot-tmpl">
   <a href="{{= MiniProfiler.shareUrl(it.page.id) }}" class="profiler-share-profiler-results" target="_blank">share</a>
   <a href="{{= MiniProfiler.moreUrl(it.timing.name) }}" class="profiler-more-actions">more</a>
+  {{? it.page.flamegraph}}
```

Comment:
This is kind of a nitpick, but I think `it.page.flamegraph` here is the actual flamegraph? Since we don't need the whole thing and just need to know if it exists, maybe we could do some magic when serializing here:

https://github.com/MiniProfiler/rack-mini-profiler/blob/aef9b2308a96be9be2236005d854a0e1d47cad00/lib/mini_profiler/timer_struct/page.rb#L114-L120

and replace `flamegraph` with a boolean `flamegraph_exists` on the fly? That way we don't have to serialize and send flamegraph data (which tends to be very large) to clients.